commons/files.go

```diff
@@ -8,7 +8,8 @@ import (
 	"regexp"
 )
 
-var hl7SplitToken = regexp.MustCompile("(\\r(\\n|\\x1c)+(\\n\\r)?|$)")
+var hl7SplitToken = regexp.MustCompile("(\\r(\\n|\\x1c)+(\\n\\r)?MSH\\||$)")
+var hl7FindStartToken = regexp.MustCompile("(MSH\\||$)")
```

Comment:
Suggestion no change to hl7SplitToken.
Change hl7FindStartToken to:
````var hl7SplitTokenVFP = regexp.MustCompile("ZXO\\r(\\n)?|$")````

Reply:
yeah that won't work, as discussed.
elaborating further on MSH might, though? e.g. ... "MSH\\|^~\\\&\\||$"?